admin/config.yml

```diff
@@ -20,3 +26,48 @@ collections:
       - {label: "Tags", name: "tags", widget: "list"}
       - {label: "Body", name: "body", widget: "markdown"}
       - {label: "Optional image for SEO", name: "image", widget: "image", required: false}
+  - name: "blog" # Used in routes, e.g., /admin/collections/blog
+    slug: '{{year}}-{{month}}-{{day}}-{{slug}}'
+    label: "Blog" # Used in the UI
+    folder: "_blog" # The path to the folder where the documents are stored
+    create: true # Allow users to create new documents in this collection
+    fields: # The fields for each document, usually in front matter
+      - {label: "Title", name: "title", widget: "string"}
+      # this is still not working...in worst case...could set it to simple string...but without validation ;p   
+      - {label: "Author", name: "author", widget: "relation", collection: "relation_files", file: "authors", search_fields: ['authors.entries.handle'], value_field: "authors.entries.handle" }
```

Comment:
@acao this line seems wrong... I am unsure what file: value should be...I defined authors in line 61 below; but it's not working :D